### ISSUE TITLE
Chevron in pagination (psuedo element font icon) should not be announced to JAWS users as "single left pointing angle quotation mark"

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/vets-design-system-documentation#434]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

JAWS will announce the chevrons by the "next" and "previous" buttons as "single right pointed angle quotation mark" which is unhelpful and verbose to screen reader users.

## What I expected to happen

Screen readers should not announce out decorative icon fonts.

## Reproducing
JAWS/Chrome, Find Forms, search "medical", use down arrow keys to reach the chevron in "next" 

## Recommendation
Would it be possible to switch to using SVG for more accessible icons instead of relying on the psuedo-element :before and :after ? [Otherwise, this article may provide a solution.](https://blog.thomaspuppe.de/accessibility-pseudo-content)

## Resources
https://www.levelaccess.com/csscontentproperty/

**Comment by jeana-adhoc** [comment]
icons in buttons are no longer being announced.  Video here shows mac Voice Over, but this was confirmed in JAWS and NVDA as well.

https://github.com/user-attachments/assets/ec99d3dc-18ef-4130-a45d-da847caaa7ba
